.flake8

```diff
@@ -81,3 +80,12 @@ per-file-ignores =
     ./posthog/settings/overrides.py: T001
     ./posthog/utils.py: T001
     ./posthog/async_migrations/test/test_migrations_not_required.py: T001
+
+    ./ee/clickhouse/materialized_columns/__init__.py: F401
```

Comment:
Could we add these ignores to the actual file? This way we make it very obvious to a developer working on these files what is applicable or not. It also means we don't need to keep this list in sync with e.g. files moving around or deleted